strawberryfields/backends/bosonicbackend/bosoniccircuit.py

```diff
@@ -281,12 +282,11 @@ def fromsmean(self, r, modes=None):
             r (array): vector of means in :math:`(x_1,p_1,x_2,p_2,\dots)` ordering
             modes (Sequence): sequence of modes corresponding to the vector of means
         """
-        mode_list = modes
         if modes is None:
-            mode_list = range(self.nlen)
+            modes = range(self.nlen)
```

Comment:
The docstring of this method might need to be updated. In particular I am not too sure about the part where `hbar=2`